### ISSUE TITLE
Would it be possible to "bundle" window buttons with app menu / tile

### ISSUE THREAD

**Issue by Pshemas** [franglais125/no-title-bar#110]
I'd love to see an option that would bundle / glue together window buttons (minimize / maximize / close) with app menu (title of an active app that is being displayed on the main GNOME bar).
Currently you have a setting to display window buttons before / after app menu or before / in / after status area. While handy it would be awesome if we could treat window controls and app menu / title as a single item so when I move "no-title-bar" to the "inside of notification area" I'd get both the buttons and window name there.
Would prevent me from accidentally closing the wrong window when I forget to look which one is really active.

BTW - love this extension. Makes GNOME noticeably better when using it with apps that have crowded UIs (3d apps for example) and every pixel counts. Hope that missing components to make it available on Wayland will land one day.

**Comment by skewty** [comment]
The [After app menu] option does that for me.

**Comment by Pshemas** [comment]
yeah, but it means you have those buttons on the top left area of the top bar, where app menu is - which IMO is not good. I'd like to have it consistent throughout the system. 
As I don't see any extension that would move app menu to the right I thought I'd ask whether bundling app menu with no-title-buttons is even doable :) .

**Comment by skewty** [comment]
Keep in mind that the AppMenu is changing in Gnome-Shell.  The menu aspect of it is basically gone in 3.32 for example.

How do get consistency if left/right tiled windows have their titlebars removed?  The controls position for one of the windows will be wrong / not intuitive / inconsistent.

And what will happen if / when quarter tiling and top / bottom tiling is supported?

I wish only maximized windows would have their title removed; not tiled windows as well.
And, I wish the titlebar slid out of the way much like the scrollbar does now.  That would give back a fair amount of vertical space on quarter tiled windows.  

Is that even possible with the current CSS implementation?
